land_vision/get_images.py

```diff
@@ -0,0 +1,51 @@
+import cv2
+import json
+import requests
+import numpy as np
+import matplotlib.pyplot as plt
+
+from mapbox import mapbox_polygon as _mapbox_polygon
+  
+
+
+def get_mapbox_static_image(latitude, longitude, zoom=15, width=400, height=300, style='streets-v11'):
+
+    base_url = f'https://api.mapbox.com/styles/v4/mapbox.{style}/static/{longitude},{latitude},{zoom}/{width}x{height}'
+    access_token = 'pk.eyJ1IjoibHVrZW0xMjMiLCJhIjoiY2ttc2U4NHkzMGd5bzJ2bG1neTBvdnRqOCJ9.o4fk2TYxHvDtEBu4ddgrmA'
```

Comment:
We can get this from config.yml, so we do not need to hardcode it. Use this: https://github.com/teal-towns/tealtowns/blob/main/mapbox/mapbox_polygon.py#L18

Reply:
Also there is already a request function to use: https://github.com/teal-towns/tealtowns/blob/main/mapbox/mapbox_polygon.py#L28
And one to get streets: https://github.com/teal-towns/tealtowns/blob/main/mapbox/mapbox_polygon.py#L295

Reply:
I shouldn't have pushed this file. This was just for me to get familiar with the mapbox API. I will delete this file.